### PR TITLE
adds package selector to civic sandbox

### PR DESCRIPTION
**Pull request by nicrocs**
WIP on setting up the package selector on the civic sandbox. 

- adds redux setup and fetches and sets sandbox state
- displays package selector after sandbox is populated

### CODE REVIEW

**Comment by DingoEatingFuzz** [comment]
By WIP do you mean I should hold off reviewing because this shouldn't go into master yet, or do you mean it's not done, but it's at least in a stable spot and can be merged?

**Comment by nicrocs** [comment]
I just mean it's not done. I think it's fine to merge as long as we understand that it's still in progress and doesn't work at this point.

File: packages/civic-sandbox/src/components/App/index.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import { css } from 'emotion';
+import Packages from '../Packages';
```
Comment (jaronheard) [inline packages/civic-sandbox/src/components/App/index.js]: what this does this do? is this necessary? @DingoEatingFuzz

Comment (nicrocs) [reply packages/civic-sandbox/src/components/App/index.js]: I should probably be clearer with my naming of that component. I named the component for the sandbox package selector `Packages`. Should probably name it `PackageSelector` or `SandboxPackageSelector`

**Pull request merged**